src/types/common.ts

```diff
@@ -0,0 +1 @@
+export type Properties = Record<string, any>;
```

Comment:
You've done a really great job! These are just a few tips for improving your skills.